src/Apps/Order2/Routes/Checkout/Order2CheckoutApp.tsx

```diff
@@ -73,38 +81,62 @@ export const Order2CheckoutApp: React.FC<Order2CheckoutAppProps> = ({
       {isLoading && <Order2CheckoutLoadingSkeleton order={order} />}
       {expressCheckoutSubmitting && <SubmittingOrderSpinner />}
       <GridColumns
+        pt={[0, "50px"]}
+        px={[0, 0, 4]}
+        backgroundColor="mono5"
```

Comment:
https://www.figma.com/design/kaBdUapJIWs9YL9ATlPDzw/Checkout-Redesign?node-id=4948-84551&m=dev
- at responsive index 3 the column view appears and this is where we add the horzontal padding `4`
- there is a point that [a gap appears below the header, but there is no second column yet](https://www.figma.com/design/kaBdUapJIWs9YL9ATlPDzw/Checkout-Redesign?node-id=4948-84551&m=dev). this is the source of `pt`.

Reply:
Non-blocking and I don't have much experience in CSS grids. Curious if it's easier to manage considering our design is not really 2 dimensional or super complex. I see the 12 column layout is provided by Palette.